app/views/layouts/_header.html.erb

```diff
@@ -4,42 +4,24 @@
     aria-controls="navbarSupportedContent-4" aria-expanded="false" aria-label="Toggle navigation">
     <span class="navbar-toggler-icon"></span>
   </button>
-  <div class="collapse navbar-collapse" id="navbarSupportedContent-4">
+  <div class="collapse navbar-collapse">
     <ul class="navbar-nav ml-auto">
+      <% if !logged_in? %>
```

Comment:
thế tại sao e k xử lý
if logged_in?
em. xử lý tính toán trả về true trc false

Reply:
tại em thấy để code html kia nhìn thuận mắt hơn ạ

Reply:
để em sửa lại